pep-0458.txt

```diff
@@ -790,10 +790,9 @@ Revoking Trust in Projects and Versions
 =======================================
 
 From time to time either a project or a version of a package will need to be revoked.
-To revoke trust in a version of a package, the bin role can simply remove the
-delegation and re-sign the bin metadata.  Similarly, an entire project may be removed
-by removing the bin metadata references to the metadata and package versions.
-All of these actions only require actions with the online bin key.
+To revoke trust in either a project or a version of a package, the associated bin-n role can 
+simply remove the delegation and re-sign the bin-n metadata.  This action only 
```

Comment:
What does *bin-n* delegate to? Shouldn't it be something like...
```suggestion
simply remove the corresponding targets and re-sign the bin-n metadata.  This action only 
```

Reply:
Agreed, good catch!